compatible/src/main/java/io/seata/rm/tcc/remoting/parser/LocalTCCRemotingParser.java

```diff
@@ -63,27 +66,37 @@ public RemotingDesc getServiceDesc(Object bean, String beanName) throws Framewor
 
     @Override
     public boolean isService(Class<?> beanClass) throws FrameworkException {
-        return isLocalTCC(beanClass);
+        return isTransactionParticipant(beanClass);
     }
 
     @Override
     public boolean isReference(Object bean, String beanName) {
         return isLocalTCC(bean);
     }
 
+    /**
+     * Determine whether there is an annotation on interface or impl {@link LocalTCC} or {@link TransactionParticipant}
+     * @param bean the bean
+     * @return boolean
+     */
     private boolean isLocalTCC(Object bean) {
         Class<?> classType = bean.getClass();
-        return isLocalTCC(classType);
+        return isTransactionParticipant(classType);
     }
 
-
-    private boolean isLocalTCC(Class<?> classType) {
+    /**
+     * Determine whether there is a transaction participant annotation ({@link LocalTCC} or {@link TransactionParticipant}) 
+     * on the given class or its interfaces
+     * @param classType the class type to check
+     * @return true if the class has either LocalTCC or TransactionParticipant annotation
+     */
+    private boolean isTransactionParticipant(Class<?> classType) {
         Set<Class<?>> interfaceClasses = ReflectionUtil.getInterfaces(classType);
         for (Class<?> interClass : interfaceClasses) {
-            if (interClass.isAnnotationPresent(LocalTCC.class)) {
+            if (interClass.isAnnotationPresent(LocalTCC.class) || interClass.isAnnotationPresent(TransactionParticipant.class)) {
```

Comment:
```
interClass.isAnnotationPresent(LocalTCC.class) || interClass.isAnnotationPresent(TransactionParticipant.class)
```
This condition determines whether it would be more comfortable to encapsulate a method separately 🤔
It seems that this conditional judgment may continue to evolve in the future. Now it appears in multiple locations. Perhaps it would be better to converge?

Reply:
> Thanks for the suggestion!
> 
> I agree we should encapsulate the annotation check logic, but I suggest handling it directly in the existing `isLocalTCC()` method rather than creating a new one. This approach has several benefits:
> 
> 1. Maintains backward compatibility
> 2. Method name still reflects its core purpose
> 3. Reduces changes needed in other code
> 
> If we need to add more annotation types in the future, we can simply extend the condition within this method.
> 
> What do you think about this approach?

This is one way, but we need to confirm whether this method needs to change its name.